CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Change Log - AWS SDK for Android
 
+## [Release 2.12.4](https://github.com/aws/aws-sdk-android/releases/tag/release_v2.12.4)
+
```

Comment:
Put this under the ### Enhancements line

Reply:
added the enhancement heading.